doc/dbus_api.md

```diff
@@ -543,6 +556,23 @@ Arguments:
 * `in b diag`: new value for the flag.
 * `out u result`: `0` if `use_diag` is correctly set for all the requested DASDs. `1` if any of the given paths is invalid (ie. it does not correspond to a known DASD), `2` in case of any other error.
 
+###### `Format` Method
+
+~~~
+Format(in  ao devices,
+       out u  result,
+       out o  job)
+~~~
+
+Starts a format process for the DASDs in the given list. It creates a job to represent such a
+process.
+
+Arguments:
+
+* `in ao devices`: paths of the D-Bus objects representing the DASDs to format.
+* `out u result`: `0` if the format operation starts correctly and the job to track it is created. `1` if any of the given paths is invalid (ie. it does not correspond to a known DASD), `2` in case of any other error.
+* `out o job`: if the result is 0, path of the new job that can be used to track the formatting. Empty string in any other case.
```

Comment:
An empty object path is not allowed, and I believe the bus would drop the connection.

The best practice for a `nil` object path is `/` (which assumes the service uses the API guidelines and does not use `/`  for its main object)

Reply:
Yes, the connection drops: https://github.com/mvidner/ruby-dbus/issues/119

Reply:
Cool. I will change that.